client-app/modules/news/components/news-article-content.vue

```diff
@@ -1,35 +1,144 @@
+  <div class="news-article-content">
+    <div class="news-article-content__title">{{ newsArticle.title }}</div>
+    <div class="news-article-content__tags">
+      <template
+        v-for="tag in newsArticle.tags"
+        :key="tag"
+      >
+        <VcChip
+          color="secondary"
+          variant="outline-dark"
+          class="news-article-content__tag"
+          clickable
+          @click="emit('tag:click', tag)"
+        >
+          {{ tag }}
+        </VcChip>
+      </template>
+    </div>
 
-    <template v-if="newsArticle.publishDate" #footer>
-      <div class="news-article-content__publish-date">{{ $d(newsArticle.publishDate) }}</div>
-    </template>
-  </VcWidget>
+    <VcMarkdownRender
+      v-if="newsArticle.contentPreview"
+      :class="isImage(newsArticle.contentPreview) ? 'news-article-content__preview-image' : 'news-article-content__preview'"
+      :src="newsArticle.contentPreview"
+    />
+
+    <div
+      v-if="newsArticle.author || newsArticle.publishDate"
+      class="news-article-content__citation"
+    >
+      <div
+        v-if="newsArticle.author"
+        class="news-article-content__citation-author"
+      >
+        <VcImage
+          v-if="newsArticle.author.iconUrl"
+          :src="newsArticle.author.iconUrl"
+          lazy
+          class="news-article-content__citation-author-image"
+        />
+
+        <span>
+          {{ $t("news.details.written-by") }} 
+          <span
+            class="news-article-content__citation-author-name"
+            @click="emit('author:click', newsArticle.author.id)"
+          >{{
+            newsArticle.author.name }}
+          </span>
+        </span>
+      </div>
+
+      <div></div>
+
+      <div
+        v-if="newsArticle.publishDate"
+        class="news-article-content__citation-publish-date"
+      >
+        {{ $t("news.details.last-updated") }} {{ $d(newsArticle.publishDate, "short") }}
+      </div>
+    </div>
+
+    <VcMarkdownRender
+      :src="newsArticle.content ?? ''"
+      class="news-article-content__content"
+    />
+  </div>
 </template>
 
-<script lang="ts" setup>
+<script
+  lang="ts"
+  setup
+>
 import { toRef } from "vue";
-import { VcWidget } from "@/ui-kit/components";
 import { VcMarkdownRender } from "@/ui-kit/components/atoms";
+import { isImage } from "../utilities";
 import type { NewsArticleContent } from "../api/graphql/types";
 
+interface IEmits {
+  (event: "tag:click", tag: string): void;
+  (event: "author:click", id: string): void;
+}
+
 interface IProps {
   newsArticle: NewsArticleContent;
 }
-const props = defineProps<IProps>();
 
+const emit = defineEmits<IEmits>();
+
+const props = defineProps<IProps>();
 const newsArticle = toRef(props, "newsArticle");
 </script>
 
 <style lang="scss">
 .news-article-content {
-  &__publish-date {
-    @apply mt-3 text-right text-sm text-neutral;
+  &__title {
+    @apply text-lg font-bold;
+  }
+
+  &__preview {
+    @apply ml-5 mr-5 text-lg text-left;
```

Comment:
Try to avoid using classes ml-..., mr-..., text-left, text-right, pl-..., pr-.... prefer to use me-..., ms-..., text-start, text-end classes. Please check it everywhere.